### ISSUE TITLE
grafana dashboard provision hang in deb systems

### ISSUE THREAD

**Issue by Vonng** [Vonng/pigsty#362]
The provision playbook hang in debian / ubuntu 

```
  - name: provisioning grafana with grafana.py
    when: inventory_hostname in groups["infra"]
    tags: dashboard_init
```

Need further investigation.

**Comment by lay-g** [comment]
May I ask if there have been any updates here?

**Comment by Vonng** [comment]
known Grafana issuse

Sqlite3 database is locked during dashboard provision. Will be fixed in the next release.

change the grafana.ini and disable unified alerting
can resolved this problem as a hot fix.

**Comment by Vonng** [comment]
![image](https://github.com/Vonng/pigsty/assets/8587410/ef0a9ff6-8444-43df-a0da-1bab24e426ff)

**Comment by Vonng** [comment]
resolved, release in v2.6
